rskj-core/src/main/java/co/rsk/peg/BridgeSerializationUtils.java

```diff
@@ -132,11 +133,15 @@ public static List<UTXO> deserializeUTXOList(byte[] data) throws IOException {
 
         int nutxos = rlpList.size();
 
-        for (int k = 0; k < nutxos; k++) {
-            byte[] utxoBytes = rlpList.get(k).getRLPData();
-            InputStream istream = new ByteArrayInputStream(utxoBytes);
-            UTXO utxo = new UTXO(istream);
-            list.add(utxo);
+        try {
```

Comment:
Just in case: this way of handling this exception breaks the loop and does not keep on processing the following if something is wrong with one element. is it ok?

Reply:
wasn't it like that before as well?

Reply:
You're right! Only I wanted to double-check. Thanks, @julia-zack !

Reply:
thanks to you :)